src/main/java/com/grud/ConnectionDb.java

```diff
@@ -0,0 +1,16 @@
+package com.grud;
+
+import lombok.SneakyThrows;
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+public class ConnectionDb {
+    private static final String url = "jdbc:mysql://localhost/freelancers_site?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC";
```

Comment:
Better move to the property file.

Reply:
what do you mean?

Reply:
https://www.mkyong.com/java/java-properties-file-examples/

Reply:
Please use a property file